src/ontology/mondo.Makefile

```diff
@@ -759,6 +759,22 @@ migrate-%: tmp/mondo-edit-%.ttl
 
 migrate: migrate-omim
 
+GARD_MAPPINGS=https://docs.google.com/spreadsheets/d/e/2PACX-1vR3AUD0NRuKsFg6dNxSGSc-zabYn57udSmajScz2_puXOEZbAmEyjyyM9508OnjoJ9Ir21-BnPAJ-KX/pub?gid=1648437389&single=true&output=tsv
```

Comment:
@matentzn You can always tag me to review these PRs if they're related to something I've done.

That is pretty cool you're able to get a TSV from a Google Sheets URL like that. Of course if you want, I can add the `robot.template` to the [GARD releases](https://github.com/monarch-initiative/gard/releases/) or we could commit it. That would be the more stable / standard way. The spreadsheet is extra work to maintain / difficult to automate.

Although I don't know how many times we'll be rerunning this part of the pipeline.

Reply:
Only once for now, after that we will use our standard system!